usr/bin/revpi-btuart

```diff
@@ -4,5 +4,15 @@
 HCIATTACH=/usr/bin/hciattach
 TTYDEVNAME="$1"
 
+# RevPi Flat: The BT_REG_ON signal is connected to the GPIO11 of the expander.
+GPIOCHIP=gpiochip3
```

Comment:
Is the gpiochip number of name fixed as *3?

Reply:
As long as we don't change the HW, this should be fixed. I don't know aother way to reference the gpio.

Reply:
"cat /sys/devices/platform/soc/3f804000.i2c/i2c-1/1-0020/name" get the chip name: tca6416
"cat /sys/devices/platform/soc/3f804000.i2c/i2c-1/1-0020/gpiochip3/dev" get the dev no: 254:3
Maybe this is the a way to ref the gpio.